settings.yml

```diff
@@ -1 +1,8 @@
 port: "_env:PORT:3000"
+database:
+  user: "_env:MSUSER:root"
+  password: "_env:MSPASS:"
+  host: "_env:MSHOST:localhost"
+  port: "_env:MSPORT:3306"
+  database: "_env:MSDATABASE:mirin"
+  poolsize: "_env:MSPOOLSIZE:5"
```

Comment:
What do you think about following the naming convention employs in our web application and prefix them with the application name?:

```
user: "_env:MIRIN_DATABASE_USER:root"
```

```
MIRIN_DATABASE_USER
MIRIN_DATABASE_PASSWORD
MIRIN_DATABASE_HOST
MIRIN_DATABASE_PORT
MIRIN_DATABASE_NAME
MIRIN_DATABASE_POOLSIZE
```